---
# You don't need to edit this file, it's empty on purpose.
# Edit theme's home layout instead if you wanna make some changes
# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
title: About me
layout: single
author_profile: true
header:
  image: /assets/images/header.jpg
---

I am a postdoctoral researcher in Evan E. Eichler's [lab](https://eichlerlab.gs.washington.edu) at the University of Washington. To learn about my dog classification system see [here](https://raw.githubusercontent.com/mrvollger/mrvollger.github.io/master/assets/images/DogClassifier.png) (credit Michelle Noyes). 

## Software

- Segemntal Duplication Assembler ([SDA](https://mrvollger.github.io/SDA/)) is for identifying and resolving collapsed SDs.  
- [SafFire](https://mrvollger.github.io/SafFire/) is for genome wide miropeats vizualizations.
- [rustybam](https://mrvollger.github.io/rustybam/) is a toolkit for processing alignments.
- [StainedGlass](https://mrvollger.github.io/StainedGlass/) makes colorful dot-plots from alignments.
- Segemntal Duplication Plots ([segDupPlots](https://github.com/mrvollger/segDupPlots)) is for visualizing resolved and unresolved SDs in a human *de novo* assembly.
- [Ideogram](https://github.com/mrvollger/ideogram) is for visualizing *de novo* assemblies against a karyotype of the human reference.


## Highlighted Publications
- Nurk S, Walenz BP, Rhie A, **Vollger MR**, Logsdon GA, Grothe R, Miga KH, Eichler EE, Phillippy AM, Koren S. 2020. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. bioRxiv. <https://doi.org/10.1101/2020.03.14.992248>.

- Hsieh, Pinghsun, **Mitchell R. Vollger**, Vy Dang, David Porubsky, Carl Baker, Stuart Cantsilieris, Kendra Hoekzema, et al. 2019. “Adaptive Archaic Introgression of Copy Number Variants and the Discovery of Previously Unknown Human Genes.” Science 366 (6463). <https://doi.org/10.1126/science.aax2083>.

- Miga, Karen H., Sergey Koren, Arang Rhie, **Mitchell R. Vollger**, Ariel Gershman, Andrey Bzikadze, Shelise Brooks, et al. 2019. “Telomere-to-Telomere Assembly of a Complete Human X Chromosome.” bioRxiv. <https://doi.org/10.1101/735928>.

- **Vollger, Mitchell R.**, Glennis A. Logsdon, Peter A. Audano, Arvis Sulovari, David Porubsky, Paul Peluso, Aaron M. Wenger, et al. 2019. “Improved Assembly and Variant Detection of a Haploid Human Genome Using Single-Molecule, High-Fidelity Long Reads.” bioRxiv. <https://doi.org/10.1101/635037>.

- **Vollger, Mitchell R.**, Philip C. Dishuck, Melanie Sorensen, Annemarie E. Welch, Vy Dang, Max L. Dougherty, Tina A. Graves-Lindsay, Richard K. Wilson, Mark J. P. Chaisson, and Evan E. Eichler. 2019. “Long-Read Sequence and Assembly of Segmental Duplications.” Nature Methods 16 (1): 88–94. <http://dx.doi.org/10.1038/s41592-018-0236-3>.


## Education 

- 2016--2021		Ph.D in Genome Sciences at University of Washington
- 2011--2015		B.S.E Computer Science at Princeton University



