\subsection{\textbf{First author}}


\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Elliott G. Swanson, Shane J. Neph, Jane Ranchalis, Katherine M. Munson, Ching-Huang Ho, et al. A haplotype-resolved view of human gene regulation. \emph{bioRxiv}. \emph{Online}. 2024. \textcolor{blue}{\href{https://doi.org/10.1101/2024.06.14.599122}{10.1101/2024.06.14.599122}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Jonas Korlach, Kiara C. Eldred, Elliott Swanson, Jason G. Underwood, Yong-Han H. Cheng, et al. Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition. \emph{bioRxiv}. \emph{Online}. 2023. \textcolor{blue}{\href{https://doi.org/10.1101/2023.09.26.559521}{10.1101/2023.09.26.559521}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Philip C. Dishuck, William T. Harvey, William S. DeWitt, Xavi Guitart, Michael E. Goldberg, et al. Increased mutation and gene conversion within human segmental duplications. \emph{Nature}. \emph{325–334,7960,617}. 2023. \textcolor{blue}{\href{https://doi.org/10.1038/s41586-023-05895-y}{10.1038/s41586-023-05895-y}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Xavi Guitart, Philip C. Dishuck, Ludovica Mercuri, William T. Harvey, Ariel Gershman, et al. Segmental duplications and their variation in a complete human genome. \emph{Science}. \emph{6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abj6965}{10.1126/science.abj6965}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Peter Kerpedjiev, Adam M Phillippy, Evan E Eichler. StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps. \emph{Bioinformatics}. \emph{Online}. 2022. \textcolor{blue}{\href{https://doi.org/10.1093/bioinformatics/btac018}{10.1093/bioinformatics/btac018}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Glennis A. Logsdon, Peter A. Audano, Arvis Sulovari, David Porubsky, Paul Peluso, et al. Improved assembly and variant detection of a haploid human genome using single‐molecule, high‐fidelity long reads. \emph{Annals of Human Genetics}. \emph{125–140,2,84}. 2019. \textcolor{blue}{\href{https://doi.org/10.1111/ahg.12364}{10.1111/ahg.12364}}}

\cvitem{\textbf{---}}{\textbf{Mitchell R. Vollger}, Philip C. Dishuck, Melanie Sorensen, AnneMarie E. Welch, Vy Dang, Max L. Dougherty, et al. Long-read sequence and assembly of segmental duplications. \emph{Nature Methods}. \emph{88--94,1,16}. 2018. \textcolor{blue}{\href{https://doi.org/10.1038/s41592-018-0236-3}{10.1038/s41592-018-0236-3}}}


\subsection{\textbf{Corresponding author}}


\cvitem{\textbf{---}}{Anupama Jha, Stephanie C. Bohaczuk, Yizi Mao, Jane Ranchalis, Benjamin J. Mallory, Alan T. Min, ... \textbf{Mitchell R. Vollger}. DNA-m6A calling and integrated long-read epigenetic and genetic analysis with fibertools. \emph{Genome Research}. \emph{gr.279095.124}. 2024. \textcolor{blue}{\href{https://doi.org/10.1101/gr.279095.124}{10.1101/gr.279095.124}}}


\subsection{\textbf{Collaborative author}}


\cvitem{\textbf{---}}{Kerry L. Bubb, Morgan O. Hamm, Joseph K. Min, Bryan Ramirez-Corona, Nicholas A. Mueth, Jane Ranchalis, \textbf{Mitchell R. Vollger}, et al. The regulatory potential of transposable elements in maize. \emph{bioRxiv}. \emph{Online}. 2024. \textcolor{blue}{\href{https://doi.org/10.1101/2024.07.10.602892}{10.1101/2024.07.10.602892}}}

\cvitem{\textbf{---}}{Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Benjamin J. Mallory, Elliott G Swanson, Jane Ranchalis, \textbf{Mitchell R. Vollger}, et al. Resolving the chromatin impact of mosaic variants with targeted Fiber-seq. \emph{bioRxiv}. \emph{Online}. 2024. \textcolor{blue}{\href{https://doi.org/10.1101/2024.07.09.602608}{10.1101/2024.07.09.602608}}}

\cvitem{\textbf{---}}{Wen-Wei Liao, Mobin Asri, Jana Ebler, Daniel Doerr, Marina Haukness, Glenn Hickey, ... \textbf{Mitchell R. Vollger}..., et al. A draft human pangenome reference. \emph{Nature}. \emph{312--324,7960,617}. 2023. \textcolor{blue}{\href{https://doi.org/10.1038/s41586-023-05896-x}{10.1038/s41586-023-05896-x}}}

\cvitem{\textbf{---}}{William S. DeWitt, Luke Zhu, \textbf{Mitchell R. Vollger}, Michael E. Goldberg, Andrea Talenti, Annabel C. Beichman, et al. mutyper: assigning and summarizing mutation types for
analyzing germline mutation spectra. \emph{Journal of Open Source Software}. \emph{5227,85,8}. 2023. \textcolor{blue}{\href{https://doi.org/10.21105/joss.05227}{10.21105/joss.05227}}}

\cvitem{\textbf{---}}{David Porubsky, \textbf{Mitchell R. Vollger}, William T. Harvey, Allison N. Rozanski, Peter Ebert, Glenn Hickey, et al. Gaps and complex structurally variant loci in phased genome assemblies. \emph{Genome Research}. \emph{496–510,4,33}. 2023. \textcolor{blue}{\href{https://doi.org/10.1101/gr.277334.122}{10.1101/gr.277334.122}}}

\cvitem{\textbf{---}}{Xiangyu Yang, Xuankai Wang, Yawen Zou, Shilong Zhang, Manying Xia, \textbf{Mitchell R. Vollger}, et al. A refined characterization of large-scale genomic differences in the first complete human genome. \emph{bioRxiv}. \emph{Online}. 2022. \textcolor{blue}{\href{https://doi.org/10.1101/2022.12.17.520860}{10.1101/2022.12.17.520860}}}

\cvitem{\textbf{---}}{Sergey Aganezov, Stephanie M. Yan, Daniela C. Soto, Melanie Kirsche, Samantha Zarate, Pavel Avdeyev, ... \textbf{Mitchell R. Vollger}..., et al. A complete reference genome improves analysis of human genetic variation. \emph{Science}. \emph{6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abl3533}{10.1126/science.abl3533}}}

\cvitem{\textbf{---}}{Nicolas Altemose, Glennis A. Logsdon, Andrey V. Bzikadze, Pragya Sidhwani, Sasha A. Langley, Gina V. Caldas, ... \textbf{Mitchell R. Vollger}..., et al. Complete genomic and epigenetic maps of human centromeres. \emph{Science}. \emph{6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abl4178}{10.1126/science.abl4178}}}

\cvitem{\textbf{---}}{Ariel Gershman, Michael E. G. Sauria, Xavi Guitart, \textbf{Mitchell R. Vollger}, Paul W. Hook, Savannah J. Hoyt, et al. Epigenetic patterns in a complete human genome. \emph{Science}. \emph{6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abj5089}{10.1126/science.abj5089}}}

\cvitem{\textbf{---}}{Savannah J. Hoyt, Jessica M. Storer, Gabrielle A. Hartley, Patrick G. S. Grady, Ariel Gershman, Leonardo G. de Lima, ... \textbf{Mitchell R. Vollger}..., et al. From telomere to telomere: The transcriptional and epigenetic state of human repeat elements. \emph{Science}. \emph{6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abk3112}{10.1126/science.abk3112}}}

\cvitem{\textbf{---}}{Sergey Nurk, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V. Bzikadze, Alla Mikheenko, \textbf{Mitchell R. Vollger}, et al. The complete sequence of a human genome. \emph{Science}. \emph{44--53,6588,376}. 2022. \textcolor{blue}{\href{https://doi.org/10.1126/science.abj6987}{10.1126/science.abj6987}}}

\cvitem{\textbf{---}}{PingHsun Hsieh, Vy Dang, \textbf{Mitchell R. Vollger}, Yafei Mao, Tzu-Hsueh Huang, Philip C. Dishuck, et al. Evidence for opposing selective forces operating on human-specific duplicated TCAF genes in Neanderthals and humans. \emph{Nature Communications}. \emph{1,12}. 2021. \textcolor{blue}{\href{https://doi.org/10.1038/s41467-021-25435-4}{10.1038/s41467-021-25435-4}}}

\cvitem{\textbf{---}}{Glennis A. Logsdon, \textbf{Mitchell R. Vollger}, PingHsun Hsieh, Yafei Mao, Mikhail A. Liskovykh, Sergey Koren, et al. The structure, function and evolution of a complete human chromosome 8. \emph{Nature}. \emph{101–107,7857,593}. 2021. \textcolor{blue}{\href{https://doi.org/10.1038/s41586-021-03420-7}{10.1038/s41586-021-03420-7}}}

\cvitem{\textbf{---}}{David Porubsky, Peter Ebert, Peter A. Audano, \textbf{Mitchell R. Vollger}, William T. Harvey, Pierre Marijon, et al. Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads. \emph{Nature Biotechnology}. \emph{Online}. 2020. \textcolor{blue}{\href{https://doi.org/10.1038/s41587-020-0719-5}{10.1038/s41587-020-0719-5}}}

\cvitem{\textbf{---}}{Wesley C. Warren, R. Alan Harris, Marina Haukness, Ian T. Fiddes, Shwetha C. Murali, Jason Fernandes, ... \textbf{Mitchell R. Vollger}..., et al. Sequence diversity analyses of an improved rhesus macaque genome enhance its biomedical utility. \emph{Science}. \emph{eabc6617,6523,370}. 2020. \textcolor{blue}{\href{https://doi.org/10.1126/science.abc6617}{10.1126/science.abc6617}}}

\cvitem{\textbf{---}}{Glennis A. Logsdon, \textbf{Mitchell R. Vollger}, Evan E. Eichler. Long-read human genome sequencing and its applications. \emph{Nature Reviews Genetics}. \emph{597--614,10,21}. 2020. \textcolor{blue}{\href{https://doi.org/10.1038/s41576-020-0236-x}{10.1038/s41576-020-0236-x}}}

\cvitem{\textbf{---}}{Sergey Nurk, Brian P. Walenz, Arang Rhie, \textbf{Mitchell R. Vollger}, Glennis A. Logsdon, Robert Grothe, et al. HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads. \emph{Genome Research}. \emph{1291--1305,9,30}. 2020. \textcolor{blue}{\href{https://doi.org/10.1101/gr.263566.120}{10.1101/gr.263566.120}}}

\cvitem{\textbf{---}}{Karen H. Miga, Sergey Koren, Arang Rhie, \textbf{Mitchell R. Vollger}, Ariel Gershman, Andrey Bzikadze, et al. Telomere-to-telomere assembly of a complete human X chromosome. \emph{Nature}. \emph{79--84,7823,585}. 2020. \textcolor{blue}{\href{https://doi.org/10.1038/s41586-020-2547-7}{10.1038/s41586-020-2547-7}}}

\cvitem{\textbf{---}}{Kishwar Shafin, Trevor Pesout, Ryan Lorig-Roach, Marina Haukness, Hugh E. Olsen, Colleen Bosworth, ... \textbf{Mitchell R. Vollger}..., et al. Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes. \emph{Nature Biotechnology}. \emph{1044--1053,9,38}. 2020. \textcolor{blue}{\href{https://doi.org/10.1038/s41587-020-0503-6}{10.1038/s41587-020-0503-6}}}

\cvitem{\textbf{---}}{Arvis Sulovari, Ruiyang Li, Peter A. Audano, David Porubsky, \textbf{Mitchell R. Vollger}, Glennis A. Logsdon, et al. Human-specific tandem repeat expansion and differential gene expression during primate evolution. \emph{Proceedings of the National Academy of Sciences}. \emph{23243--23253,46,116}. 2019. \textcolor{blue}{\href{https://doi.org/10.1073/pnas.1912175116}{10.1073/pnas.1912175116}}}

\cvitem{\textbf{---}}{PingHsun Hsieh, \textbf{Mitchell R. Vollger}, Vy Dang, David Porubsky, Carl Baker, Stuart Cantsilieris, et al. Adaptive archaic introgression of copy number variants and the discovery of previously unknown human genes. \emph{Science}. \emph{6463,366}. 2019. \textcolor{blue}{\href{https://doi.org/10.1126/science.aax2083}{10.1126/science.aax2083}}}

\cvitem{\textbf{---}}{Flavia A. M. Maggiolini, Stuart Cantsilieris, Pietro D'Addabbo, Michele Manganelli, Bradley P. Coe, Beth L. Dumont, ... \textbf{Mitchell R. Vollger}..., et al. Genomic inversions and GOLGA core duplicons underlie disease instability at the 15q25 locus. \emph{PLOS Genetics}. \emph{e1008075,3,15}. 2019. \textcolor{blue}{\href{https://doi.org/10.1371/journal.pgen.1008075}{10.1371/journal.pgen.1008075}}}


